plexapi/media.py

```diff
@@ -170,52 +176,74 @@ class MediaPartStream(PlexObject):
             server (:class:`~plexapi.server.PlexServer`): PlexServer object this is from.
             initpath (str): Relative path requested when retrieving specified data.
             part (:class:`~plexapi.media.MediaPart`): Media part this stream belongs to.
+            bitrate (int):
             codec (str): Codec of this stream (ex: srt, ac3, mpeg4).
-            codecID (str): Codec ID (ex: XVID).
+            default (str):
+            displayTitle (str):
+            extendedDisplayTitle (str):
+            key (str):
             id (int): Unique stream ID on this server.
             index (int): Unknown
             language (str): Stream language (ex: English, ไทย).
             languageCode (str): Ascii code for language (ex: eng, tha).
+            requiredBandwidths (str):
             selected (bool): True if this stream is selected.
             streamType (int): Stream type (1=:class:`~plexapi.media.VideoStream`,
                 2=:class:`~plexapi.media.AudioStream`, 3=:class:`~plexapi.media.SubtitleStream`).
+            title (str):
             type (int): Alias for streamType.
     """
 
     def _loadData(self, data):
         """ Load attribute values from Plex XML response. """
         self._data = data
+        self.audioProfile = data.attrib.get('audioProfile')
+        self.bitrate = cast(int, data.attrib.get('bitrate'))
         self.codec = data.attrib.get('codec')
-        self.codecID = data.attrib.get('codecID')
+        self.deepAnalysisVersion = cast(int, data.attrib.get('deepAnalysisVersion'))
+        self.default = data.attrib.get('default')
+        self.displayTitle = data.attrib.get('displayTitle')
+        self.extendedDisplayTitle = data.attrib.get('extendedDisplayTitle')
+        self.key = data.attrib.get('key')
+        self.has64bitOffsets = cast(bool, data.attrib.get('has64bitOffsets'))
+        self.hasThumbnail = cast(bool, data.attrib.get('hasThumbnail'))
         self.id = cast(int, data.attrib.get('id'))
         self.index = cast(int, data.attrib.get('index', '-1'))
         self.language = data.attrib.get('language')
         self.languageCode = data.attrib.get('languageCode')
+        self.packetLength = cast(int, data.attrib.get('packetLength'))
+        self.requiredBandwidths = data.attrib.get('requiredBandwidths')
```

Comment:
`packetLength` should be under `MediaPart`.
`requiredBandwidths` should be under `MediaPart`.